### PR TITLE
Fix REST API tests (import with timeout)

### PR DESCRIPTION
**Pull request by yasakova-anastasia**
### Checklist

- [ ] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into the [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

**Comment by azhavoro** [comment]
It seems that the reason for the this test fails is default value of the check interval for the scheduler. I'm ok with this as a quick fix, but it looks better if we use a lower interval value for testing purposes only in the future without increasing the test timeout.

https://github.com/rq/rq-scheduler/blob/master/rq_scheduler/scheduler.py#L31
 https://github.com/rq/django-rq/blob/master/django_rq/queues.py#L329

**Comment by yasakova-anastasia** [comment]
@azhavoro, could you please review again?

File: cvat/settings/testing_rest.py [hunk 1]
```diff
@@ -11,4 +11,6 @@
     "django.contrib.auth.hashers.MD5PasswordHasher",
 ]
 
-IMPORT_CACHE_CLEAN_DELAY = timedelta(seconds=30)
+IMPORT_CACHE_CLEAN_DELAY = timedelta(seconds=1)
```
Comment (Marishka17) [inline cvat/settings/testing_rest.py:14]: Why did you change it to 1 second? It means that the cleaning job will start after 1 second after the file upload was finished, not imported. I guess that the tests are passed now because the interval is more than IMPORT_CACHE_CLEAN_DELAY. Logically it looks wrong to me.

File: tests/python/rest_api/test_tasks.py [hunk 2]
```diff
@@ -1825,7 +1825,7 @@ def test_check_import_cache_after_previous_interrupted_upload(self, tasks_with_s
             url, filename, meta=params, logger=self.client.logger.debug
         )
         number_of_files = 1
-        sleep(30)  # wait when the cleaning job from rq worker will be started
+        sleep(5)  # wait when the cleaning job from rq worker will be started
```
Comment (Marishka17) [inline tests/python/rest_api/test_tasks.py:1828]: Why do we need to sleep only 5 seconds here?
How about changing the number of iterations in `for _ in range(12):` too?

Comment (azhavoro) [reply tests/python/rest_api/test_tasks.py]: I suppose this value should be the same as `RQ_SCHEDULER_CHECK_INTERVAL + IMPORT_CACHE_CLEAN_DELAY`, it also seems that we can reduce `RQ_SCHEDULER_CHECK_INTERVAL` to 5-10s

File: cvat/settings/testing_rest.py (hunk 1 above)
Comment (azhavoro) [reply cvat/settings/testing_rest.py]: I think we can set 15s here for two reasons:
1. we don't currently have huge test data to import
2. 15 is the standard timeout for test cases

File: tests/python/rest_api/test_tasks.py (hunk 2 above)
Comment (azhavoro) [reply tests/python/rest_api/test_tasks.py]: @Marishka17 please correct me if I'm wrong

File: cvat/settings/testing_rest.py (hunk 1 above)
Comment (yasakova-anastasia) [reply cvat/settings/testing_rest.py]: Set 15s for IMPORT_CACHE_CLEAN_DELAY and 10s for scheduler interval.

**Review by Marishka17 — APPROVED** [review]

**Comment by Marishka17** [comment]
@yasakova-anastasia, Could you please resolve conflicts?

**Comment by azhavoro** [comment]
Unfortunately, the test still fails: https://github.com/opencv/cvat/actions/runs/5240951371/jobs/9472316010
I'm starting to think that the problem might not be the timeout setting...

File: cvat/apps/dataset_manager/views.py [hunk 3]
```diff
@@ -85,7 +85,10 @@ def export(dst_format, project_id=None, task_id=None, job_id=None, server_url=No
                 os.replace(temp_file, output_path)
 
             archive_ctime = osp.getctime(output_path)
-            scheduler = django_rq.get_scheduler(settings.CVAT_QUEUES.EXPORT_DATA.value)
+            scheduler = django_rq.get_scheduler(
+                settings.CVAT_QUEUES.EXPORT_DATA.value,
+                interval=settings.DJANGO_RQ_SCHEDULER_INTERVAL
```
Comment (zhiltsov-max) [inline cvat/apps/dataset_manager/views.py:90]: Are you sure the interval should be set here and not in the scheduler startup parameters?
